### PR TITLE
INV-170: added endpoint for new AccountListing report

### PR DESCRIPTION
**Pull request by duyhoan**
This MR is part of the ticket [INV-141](https://hitmeister.atlassian.net/browse/INV-141). Goals of the ticket:
- Integrate new API endpoint to API SDK. This new endpoint is used to create new `AccountListing` report, which includes the current prices of item on Marketplace.

Closing INV-170.

### CODE REVIEW

File: src/Endpoints/Reports/AccountListingWithListingPrice.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+class AccountListingWithListingPrice extends Post
+{
+	/**
+	 * {@inheritdoc}
+	 */
+	public function getURI()
+	{
+		return 'reports/account-listing-with-listing-price/';
+	}
+}
```
Comment (aazon) [inline src/Endpoints/Reports/AccountListingWithListingPrice.php]: Please add a new line at the end of this file

File: tests/Endpoints/Reports/AccountListingWithListingPriceTest.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+class AccountListingWithListingPriceTest extends TransportAwareTestCase
+{
+	public function testInstance()
+	{
+		$post = new AccountListingWithListingPrice($this->transport);
+		$this->assertEquals([], $post->getParamWhiteList());
+		$this->assertEquals('POST', $post->getMethod());
+		$this->assertEquals('reports/account-listing-with-listing-price/', $post->getURI());
+	}
+}
```
Comment (aazon) [inline tests/Endpoints/Reports/AccountListingWithListingPriceTest.php]: Please add a new line at the end of this file as well

File: src/Endpoints/Reports/AccountListingWithListingPrice.php (hunk 1 above)
Comment (duyhoan) [reply src/Endpoints/Reports/AccountListingWithListingPrice.php]: added new line at the end

File: tests/Endpoints/Reports/AccountListingWithListingPriceTest.php (hunk 2 above)
Comment (duyhoan) [reply tests/Endpoints/Reports/AccountListingWithListingPriceTest.php]: added new line at the end

**Review by aazon — APPROVED** [review]

**Review by NastiaNovik — CHANGES_REQUESTED** [review]
Please use tabs instead of spaces, because it is our guideline

**Review by NastiaNovik — APPROVED** [review]

**Pull request merged**